addons/hr_timesheet/models/res_company.py

```diff
@@ -10,31 +10,19 @@ class ResCompany(models.Model):
 
     @api.model
     def _default_project_time_mode_id(self):
-        uom = self.env.ref('uom.product_uom_hour', raise_if_not_found=False)
-        wtime = self.env.ref('uom.uom_categ_wtime')
-        if not uom:
-            uom = self.env['uom.uom'].search([('category_id', '=', wtime.id), ('uom_type', '=', 'reference')], limit=1)
-        if not uom:
-            uom = self.env['uom.uom'].search([('category_id', '=', wtime.id)], limit=1)
-        return uom
+        return self.env.ref('uom.product_uom_hour', raise_if_not_found=False)
 
     @api.model
     def _default_timesheet_encode_uom_id(self):
-        uom = self.env.ref('uom.product_uom_hour', raise_if_not_found=False)
-        wtime = self.env.ref('uom.uom_categ_wtime')
-        if not uom:
-            uom = self.env['uom.uom'].search([('category_id', '=', wtime.id), ('uom_type', '=', 'reference')], limit=1)
-        if not uom:
-            uom = self.env['uom.uom'].search([('category_id', '=', wtime.id)], limit=1)
-        return uom
-    
+        return self.env.ref('uom.product_uom_hour', raise_if_not_found=False)
```

Comment:
Should be double checked later on that this doesn't break anything (i.e. make it so that no uom is assigned) since `project_time_mode_id` and `timesheet_encode_uom_id` aren't required